src/main/java/tech/gruppone/stalkerserver/configuration/ApplicationConfiguration.java

```diff
@@ -1,20 +1,118 @@
 package tech.gruppone.stalkerserver.configuration;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.influxdb.InfluxDB;
+import org.influxdb.dto.Point;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.annotation.PropertySources;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
-// TODO test that configuration works as expected
+//TODO test that configuration works as expected
 //TODO should eventually use configuration annotation
 //FIXME should not be a RestController
 //@Configuration
-@PropertySource("classpath:application.properties")
+@CrossOrigin
+@PropertySources({
+  @PropertySource("classpath:application.properties"),
+  @PropertySource("classpath:local.properties")
+})
 @RestController
 public class ApplicationConfiguration {
 
+  @Autowired
+  public InfluxDB influxDB;
+
+  public InfluxDBPoint point = new InfluxDBPoint();
+
+  @RequestMapping("/")
+  public String index(){
+    return "localhost:8080/ works well!";
+  }
+
```

Comment:
questa roba va in un package chiamato

package tech.gruppone.stalkerserver.location;

con nome della classe LocationUpdateController